src/features/DisplayControls/NursingTasks/components/NursingTasks.jsx

```diff
@@ -13,15 +13,22 @@ import TaskTile from "./TaskTile";
 import { formatDate } from "../../../../utils/DateTimeUtils";
 import { SliderContext } from "../../../../context/SliderContext";
 import UpdateNursingTasks from "./UpdateNursingTasks";
-import { Button } from "carbon-components-react";
+import { Button, Dropdown } from "carbon-components-react";
 import AddEmergencyTasks from "./AddEmergencyTasks";
 
 export default function NursingTasks(props) {
   const { patientId } = props;
   const [medicationNursingTasks, setMedicationNursingTasks] = useState([]);
+  const [nursingTasks, setNursingTasks] = useState([]);
   const [isLoading, setIsLoading] = useState(true);
   const { isSliderOpen, updateSliderOpen } = useContext(SliderContext);
   const [selectedMedicationTask, setSelectedMedicationTask] = useState([]);
+  const items = [
```

Comment:
can we move this to the nursing task constants or utils file?

Reply:
Okay sure